src/app/components/link-custom/link-custom.module.scss

```diff
@@ -0,0 +1,130 @@
+.button {
+  @extend %transition-all;
+
+  cursor: pointer;
+
+  display: flex;
+  flex-flow: row nowrap;
+  gap: 8px;
+  place-content: center center;
+  align-items: center;
+
+  font-weight: $font-weight-medium;
+  letter-spacing: 0;
+  white-space: pre;
+}
+
+.colored {
+  width: min-content;
+
+  &.large {
+    @include media-tablet {
+      width: 100%;
+    }
+
+    overflow: visible;
+
+    margin-top: 24px;
+    padding: 16px 24px;
+
+    font-size: 18px;
+    line-height: 24px;
+    color: $color-white;
+
+    background-color: $color-black;
+
+    &:hover {
+      background-color: $color-gray-600;
+    }
+  }
+
+  &.small {
+    @include media-tablet {
+      gap: 0;
+
+      &.regular {
+        margin-right: 8px;
+      }
+    }
+
+    overflow: hidden;
+
+    height: 36px;
+    padding: 10px 12px;
+
+    font-size: 14px;
+    line-height: 16px;
+    color: $color-black;
+
+    background-color: #eee;
+    border-radius: 9999px;
+
+    &:hover {
+      background-color: hsla(from $color-gray-600 h s l/ $opacity-50);
+    }
+  }
+}
+
+.outlined {
+  margin: 24px 0;
+  color: $color-black;
+  border: 2px solid $color-black;
+
+  &.large {
+    width: max-content;
+    padding: 12px 24px;
+    font-size: 18px;
+
+    &:hover {
+      background-color: $color-gray-100;
+    }
+
+    &.rounded {
```

Comment:
Is there a reason why only large buttons can be rounded? Because if I use ```<LinkCustom href={href} button size="medium" outlined target="_blank">```, it won't work

Reply:
I'm afraid medium or small buttons could have another styles